pkg/publish/servicelb.go

```diff
@@ -45,8 +48,14 @@ func ServiceLoadBalancer(req router.Request, svc *v1.ServiceInstance) (result []
 		svc.Spec.Annotations = map[string]string{}
 	}
 
-	for _, entry := range typed.Sorted(cfg.ServiceLBAnnotations) {
-		svc.Spec.Annotations[entry.Key] = entry.Value
+	sort.Strings(cfg.ServiceLBAnnotations)
```

Comment:
No need to sort the slice. The result will be consistent because the items in the slice come out in the same order each time.